### PR TITLE
Two extract functions: extract.bife, extract feglm

### PR DESCRIPTION
**Pull request by czucca**

### CODE REVIEW

**Comment by czucca** [comment]
the third function suggested by Christoph Riedl, `extract.felm` is already in the package.

**Comment by czucca** [comment]
actually I relized now there was already a PR for feglm...

File: R/extract.R [hunk 1]
```diff
@@ -1,5152 +1,5255 @@
...
-  }
-  if (include.bic == TRUE) {
-    gof <- c(gof, bic)
-    gof.names <- c(gof.names, "BIC")
-    gof.decimal <- c(gof.decimal, TRUE)
-  }
-  if (include.loglik == TRUE) {
-    gof <- c(gof, lik)
-    gof.names <- c(gof.names, "Log Likelihood")
-    gof.decimal
```
Comment (leifeld) [inline R/extract.R]: Remove blank line.

Comment (leifeld) [inline R/extract.R]: Remove blank line.

Comment (leifeld) [inline R/extract.R]: Add spaces around `=`.

Comment (leifeld) [inline R/extract.R]: `names` is an `R` function. It's always a bad idea to overwrite functions. Can you find a different name for this object?

Comment (leifeld) [inline R/extract.R]: It strikes me that this function looks very much like many other functions. Can you check if there is an identical function somewhere in the file? If that's the case, we can just create a copy and can avoid all the duplication...

Comment (leifeld) [inline R/extract.R]: Remove blank line.

Comment (leifeld) [inline R/extract.R]: Remove blank line.

Comment (leifeld) [inline R/extract.R]: Fix spaces.

Comment (leifeld) [inline R/extract.R]: Don't overwrite `names`. See above.

File: vignettes/texreg.Rnw [hunk 2]
```diff
@@ -1,1343 +1,1345 @@
...
-extract.lm <- function(model) {
-  s <- summary(model)
-  names <- rownames(s$coef)
-  co <- s$coef[, 1]
-  se <- s$coef[, 2]
-  pval <- s$coef[, 4]
-
-  rs <- s$r.squared
-  adj <- s$adj.r.squared
-  n <-
```
Comment (leifeld) [inline vignettes/texreg.Rnw]: Increase more if necessary because new models have been added in the meantime.

Comment (leifeld) [inline vignettes/texreg.Rnw]: The spacing here looks odd.

**Comment by czucca** [comment]
Hi Philip, 

I checked whether extract.bife was compatible with other functions. Even if it is really similar to the glm family, it is not compatible for gofs, which are extracted differently. So I kept it the way it was after fixing spacing and coefficient names.

File: vignettes/texreg.Rnw [hunk 3]
```diff
@@ -439,16 +439,18 @@ Class			& Package			& Description \\
 \bottomrule
 \endfoot
 
-\caption[List of 96 supported model types (version 1.37).]{\label{tab:types} List of 96 supported model types (version 1.37).} \\
+\caption[List of 99 supported model types (version 1.36.24).]{\label{tab:types} List of 99 supported model types (version 1.36.24).} \\
 \endlastfoot
-`\code{aftreg}'			& \pkg{eha}			& Accelerated failure time regression \\
+
+`\code{aftreg}'			& \pkg{eha}			    & Accelerated failure time regression \\
```
Comment (leifeld) [inline vignettes/texreg.Rnw]: Can you check the spacing in this table please?

Comment (czucca) [reply vignettes/texreg.Rnw]: yeah, sorry, I did it already, but as I resolved conflicts it got messed up again. I'll do it for the second time

**Comment by leifeld** [comment]
Thanks! Feel free to merge when done! :)

**Comment by czucca** [comment]
Thanks for letting me merge ^_^

**Pull request merged**